D3D12Bundles/D3D12Bundles.cs

```diff
@@ -541,30 +543,30 @@ struct Vertex {
 
                     ushort mipLevels = 1;
                     textureDesc = ResourceDescription.Texture2D(format, (uint)firstFrame.PixelWidth, (uint)firstFrame.PixelHeight, 1, mipLevels, 1, 0, ResourceFlags.None);
-                    mTexture = mDevice.CreateCommittedResource(HeapProperties.DefaultHeapProperties, HeapFlags.None,
-                                                               textureDesc, ResourceStates.CopyDest, null);
-                    mTexture.Name = nameof(mTexture);
+                    var textureResource = mGraphicsDevice.NativeDevice.CreateCommittedResource(HeapProperties.DefaultHeapProperties, HeapFlags.None,
+                                                                                               textureDesc, ResourceStates.CopyDest, null);
+                    mTexture = new Texture(mGraphicsDevice, textureResource, nameof(mTexture));
```

Comment:
Minor change, we simply use a Texture class to encapsulate the original resource for compatibility with the other higher level abstractions we're moving to.... it still exposes the original resource via its `NativeResource` property, but we can now use it with the likes of `CommandList.ResourceBarrierTransition` too.